modules/HTML.tla

```diff
@@ -0,0 +1,208 @@
+-------------------------------- MODULE HTML --------------------------------
+
+LOCAL INSTANCE Sequences
+LOCAL INSTANCE SequencesExt
+
+\***** Helpers
+
+StringFill(string, args) ==
```

Comment:
Add `LOCAL` to helper/internal operators?

Reply:
I also use those operators also outside of the html module

StringFill here: https://github.com/afonsonf/tlaplus-html-examples/blob/main/examples/example-4/MissionariesAndCannibals_Template.tla#L43

StringSeqToString here: https://github.com/afonsonf/tlaplus-html-examples/blob/main/examples/example-2/EWD998_Template.tla#L19

Reply:
I'd suggest disambiguating TLA+ strings from the "String" here or using the operators in `SequencesExt`.

Reply:
What if I rename the `StringFill` to `HTMLFill`? 

For `StringSeqToString` I think I can replace it by `FlattenSeq`

Reply:
Done